target/disable.ps1

```diff
@@ -65,11 +65,12 @@ try {
         'DisableAccount' {
             if (-not($actionContext.DryRun -eq $true)) {
                 Write-Information "Disabling {connectorName} account with accountReference: [$($actionContext.References.Account)]"
-                # < Write disable logic here >
 
                 if ($actionContext.Origin -eq 'reconciliation') {
                     # During reconciliation, hardcoded values may need to be set as personContext and actionContext.Data are not available
                     # < Write reconciliation disable logic here >
+                } else {
+                    # < Write disable logic here >
                 }
```

Comment:
Op zich is dit ok. Echter, kun je niet helemaal voorkomen dat iemand toch direct onder de if op regel 66, disable logic gaat schrijven. Misschien is het beter om hier een switch toe te passen op `$actionContext.Origin`. 
```suggestion
                if ($actionContext.Origin -eq 'reconciliation') {
                    # During reconciliation, hardcoded values may need to be set as personContext and actionContext.Data are not available
                    # < Write reconciliation disable logic here >
                } else {
                    # < Write disable logic here >
                }
```